scripts/instances/deploy-dhis2-core.sh

```diff
@@ -22,6 +22,8 @@ CHART_VERSION=${CHART_VERSION:-0.12.1}
 INSTANCE_TTL=${INSTANCE_TTL:-""}
 FLYWAY_MIGRATE_OUT_OF_ORDER=${FLYWAY_MIGRATE_OUT_OF_ORDER:-false}
 FLYWAY_REPAIR_BEFORE_MIGRATION=${FLYWAY_REPAIR_BEFORE_MIGRATION:-false}
+CORE_RESOURCES_REQUESTS_CPU=${CORE_RESOURCES_REQUESTS_CPU:-250m}
```

Comment:
Since we're already in the context of core maybe strip the core part of the name

Reply:
Sure 👍